### PR TITLE
you can examine the state of miners now

### LINKED ISSUE #27985: gas miners don't tell you if they're on or off when examined
#### Description of issue

you can look at the sprite but if you can't see it for whatever reason there's no other way to figure out if the damn thing is pumping or not

### PR DESCRIPTION
**Pull request by jjpark-kb**
closes: #27985
easy fix they said, well maybe I don't want an easy fix and make it better!
:cl:
 * rscadd: Gas miners will now tell you their operating state when examined. No power? On? Broken? the miner will tell you

### CODE REVIEW

**Comment by jjpark-kb** [comment]
I guess qol would be an appropriate tag as well

edit: and tested as well.
image
image
image

File: code/game/machinery/atmoalter/gas_mine.dm [hunk 1]
```diff
@@ -30,6 +32,21 @@
...
+		if(2)
+			examine_flavor = "Turned off"
+		if(3)
+			examine_flavor = "Broken"
+		if(4)
+			examine_flavor = "Operating normally"
+
+/obj/machinery/atmospherics/miner/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='info'>\The [src]'s debug reads: [examine_flavor].</span>")
```
Comment (DamianX) [inline code/game/machinery/atmoalter/gas_mine.dm]: The gas miner's debug? What's a debug?

**Review by DamianX — CHANGES_REQUESTED** [review]
This approach works, but it would be better for `examine()` to perform the checks on the spot, rather than continuously update this new `examine_flavor` var that will more often than not be unutilized. This doesn't really matter at the end of the day, but it would at least be consistent with how the rest of the game is implemented.

**Comment by press-mark** [comment]
why not move flavor_examine into the examine proc?

**Pull request merged**